include/qpdf/ObjectHandle.hh

```diff
@@ -44,6 +50,8 @@ namespace qpdf
 
         // The rest of the header file is for qpdf internal use only.
 
+        inline qpdf_object_type_e type_code() const;
```

Comment:
... or is this supposed to be protected? I would think this is public so should be above the comment.

Reply:
They are part of the private API. The implementation is in a private header and not accessible externally.

Reply:
What is the purpose of a public inline function whose implementation is private? I guess it just makes it accessible everywhere within the internals? I guess it's okay, but it seems a little weird. I guess I'm murky on this choice. I don't have a problem with it necessarily as a declaration of an inline function with no implementation shouldn't have any affect on the ABI.